src/components/Button/Button.module.css

```diff
@@ -1,14 +1,13 @@
 .base {
-    border-radius: var(--button-border-radius);
-    line-height: var(--button-line-height);
+    border-radius: var(--border-radius-small);
+    line-height: 32px;
     font-size: 14px;
     padding: 0 16px;
```

Comment:
I see that the height of the input and button is different. I also see that the input is taller than it is in Figma. So my bad.

Buttttt, I like how the input looks right now the way it is developed. Can you make the button the same height as the input (39px). Sorry i have mental issues 🙃

Reply:
39?? That's an odd number 😋

Reply:
@RobinWijnant what do you think about 36px as a general height for button and input? Meet in the middle